apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/ResponseField.kt

```diff
@@ -304,34 +304,6 @@ open class ResponseField internal constructor(
       )
     }
 
-    /**
-     * Factory method for creating a Field instance representing [Type.LONG].
-     *
-     * @param responseName alias for the result of a field
-     * @param fieldName name of the field in the GraphQL operation
-     * @param arguments arguments to be passed along with the field
-     * @param optional whether the arguments passed along are optional or required
-     * @param conditions list of conditions for this field
-     * @return Field instance representing [Type.LONG]
-     */
-    @JvmStatic
-    fun forLong(
```

Comment:
Since these are in the api, is it fine to remove without deprecating them first?

Reply:
3.x will break a huge amount of things. I don't really see a nice deprecation path there. We could deprecate in `2.x` but deprecating without giving a clear path forward isn't great either.

Reply:
Makes sense :+1: